frontends/PyCDE/src/pycde/system.py

```diff
@@ -2,7 +2,8 @@
 #  See https://llvm.org/LICENSE.txt for license information.
 #  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-from pycde.devicedb import EntityExtern, PlacementDB, PrimitiveDB, PhysicalRegion
+from pycde.devicedb import (EntityExtern, PlacementDB, PrimitiveDB,
+                            PhysicalRegion)
```

Comment:
Is this a different auto-save or something? Does the format need to change here? If so, can you please do a big re-format first and commit separately?

Reply:
No, this was a one-off change which I made. IIRC I was importing something new (since removed) but the line is 81 characters in main, so I kept this. But you're right...